index.html

```diff
@@ -6,60 +6,62 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="style.css">
     <title>Assignment 1</title>
+    <script src="login.js"></script>
 </head>
 
 <body>
     <main>
         <div class="container">
             <div class="container__left">
-                <div class="info__area">
-                    <p class="info__area--h1">
+                <div class="info-area">
+                    <p class="info-area__h1">
                         comproDLS<span>TM</span>
                     </p>
-                    <p class="info__area--h2">
+                    <p class="info-area__h2">
                         OnBoarding
                     </p>
-                    <hr id="info__area--line">
-                    <p class="info__area--h3">
+                    <hr id="info-area__hr">
+                    <p class="info-area__h3">
                         Application
                     </p>
-                    <p class="info__area--text">
+                    <p class="info-area__text mt-2">
                         Welcome to the comproDLS<span>TM</span> OnBoarding 
                         Application for mentors and freshers. Please sign in using your organization email and password.
-                        <br><br>
+                    </p>
+                    <p class="info-area__text mt-1">
                         In case you do not have an account, click on 'Sing Up now' to get started.
                     </p>
                 </div>
             </div>
             <div class="container__right">
-                <div class="login__area">
+                <div class="login-area">
                     <p class="signup web-visi">
```

Comment:
we can use a single class instead of `signup web-visi` unless one of them is used at more than 1 places.